web/libs/datamanager/src/components/Filters/types/List.jsx

```diff
@@ -14,9 +17,21 @@ export const VariantSelect = observer(({ filter, schema, onChange, multiple, val
   })();
 
   const FilterItem = filter.cellView?.FilterItem;
+
+  const filteredItems = useMemo(() => {
+    if (filter.field?.alias === "annotators" || filter.field?.alias === "reviewers") {
+      const itemsArray = items?.toJSON ? items.toJSON() : items;
+      return itemsArray?.filter((item) => {
+        const user = sdk.store.users.find((u) => u.id === item);
```

Comment:
Is `item` guaranteed to always be an id, also does `sdk.store.users` already have the full list possible for the org prior to this dropdown being accessed?

Next thought, in the cases of a very large number of users, this is running possibly `n^2` or `mXn` depending on whether the itemsArray is the full list of user ids or not.

Reply:
Also, this all feels like something that should be lifted out of here and closer to the implementation point where it is needed, this code looks very agnostic to record types.